source/includes/read/specify-queries.rb

```diff
@@ -0,0 +1,79 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+    # start-setup
+    database = client.use('db')
```

Comment:
I don't know if it matters for these demonstration scripts, but the DBX Ruby team has standardized on 2-space indentations.

```suggestion
  # start-setup
  database = client.use('db')
```